tests/__init__.py

```diff
@@ -25,12 +25,15 @@ def makeUnitSquareWire():
     )
 
 
-def makeUnitCube():
-    return makeCube(1.0)
+def makeUnitCube(centered=True):
```

Comment:
If a unit cube is supposed to be centered, should it be centered on the Z axis too so that the center of the cube is at the origin (0, 0, 0)? The current method only centers on the X and Y. If that's the way it was before, just disregard this comment.

Reply:
It was not centered at all, I'll adjust it.

Reply:
I changed the param to `xycentered`. Complete centering was breaking more tests.